Kernel/Graphics/Intel/VirtualMemory/GPUVirtualAddress.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Format.h>
+#include <AK/Types.h>
+
+namespace Kernel::IntelGraphics {
+
+class GPUVirtualAddress {
```

Comment:
This class looks like it's doing a lot of heavy lifting that it doesn't need to. Could we create it like this instead?
```
class GPUVirtualAddress : public VirtualAddress
{
   using VirtualAddress::VirtualAddress;
};
```
If not... there's a couple more comments.